src/amo/components/UserProfileEdit/index.js

```diff
@@ -328,6 +328,7 @@ export class UserProfileEditBase extends React.Component<Props, State> {
       notifications,
       occupation,
       picture,
+      pictureData,
```

Comment:
This code would make more sense to me if these were named `pictureURL` and `pictureDataURL` but no need to change them now -- just an observation.

Reply:
I don't think these information are URLs _per se_ though.